components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/GleanLifecycleObserver.kt

```diff
@@ -24,6 +24,10 @@ internal class GleanLifecycleObserver : LifecycleObserver {
         // on foreground.
         Baseline.duration.stopAndSum()
         Glean.handleEvent(Glean.PingEvent.Background)
+
+        // This will check the metrics ping schedule to determine whether it's time to send a new
+        // metrics ping or not
+        Glean.handleEvent(Glean.PingEvent.Default)
```

Comment:
Let's also add a test, similarly to how it's [done here](https://github.com/mozilla-mobile/android-components/blob/74f0ae6e7ab84f18222518aadb445c51ea0af153/components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt#L196-L206), to make sure this line is covered. This should also make sure that a `metrics` ping is generated and that the expected data is there. To do so, let's create a sample metric with Ping lifetime, record it, and make sure it's collected.